src/sizzle.js

```diff
@@ -1161,1122 +1209,1181 @@ Expr = Sizzle.selectors = {
 				7 sign of y-component
 				8 y of y-component
 			*/
-			match[1] = match[1].toLowerCase();
+				match[ 1 ] = match[ 1 ].toLowerCase();
 
-			if ( match[1].slice( 0, 3 ) === "nth" ) {
-				// nth-* requires argument
-				if ( !match[3] ) {
-					Sizzle.error( match[0] );
-				}
+				if ( match[ 1 ].slice( 0, 3 ) === "nth" ) {
 
-				// numeric x and y parameters for Expr.filter.CHILD
-				// remember that false/true cast respectively to 0/1
-				match[4] = +( match[4] ? match[5] + (match[6] || 1) : 2 * ( match[3] === "even" || match[3] === "odd" ) );
-				match[5] = +( ( match[7] + match[8] ) || match[3] === "odd" );
+					// nth-* requires argument
+					if ( !match[ 3 ] ) {
+						Sizzle.error( match[ 0 ] );
+					}
 
-			// other types prohibit arguments
-			} else if ( match[3] ) {
-				Sizzle.error( match[0] );
-			}
+					// numeric x and y parameters for Expr.filter.CHILD
+					// remember that false/true cast respectively to 0/1
+					match[ 4 ] = +( match[ 4 ] ? match[ 5 ] + ( match[ 6 ] || 1 ) : 2 *
+						( match[ 3 ] === "even" || match[ 3 ] === "odd" ) );
+					match[ 5 ] = +( ( match[ 7 ] + match[ 8 ] ) || match[ 3 ] === "odd" );
 
-			return match;
-		},
+					// other types prohibit arguments
+				} else if ( match[ 3 ] ) {
+					Sizzle.error( match[ 0 ] );
+				}
 
-		"PSEUDO": function( match ) {
-			var excess,
-				unquoted = !match[6] && match[2];
+				return match;
+			},
 
-			if ( matchExpr["CHILD"].test( match[0] ) ) {
-				return null;
-			}
+			"PSEUDO": function( match ) {
+				var excess,
+					unquoted = !match[ 6 ] && match[ 2 ],
+					unqotedLength = unquoted.length;
```

Comment:
Please don't cache this variable. This PR shouldn't change code in Sizzle source, only formatting.

Reply:
You are right